server/apiProxy.ts

```diff
@@ -193,6 +169,32 @@ export const proxyApiHandler =
 			});
 	};
 
+export const authorizationOrCookieHeader = async ({
+	req,
+	host,
+}: {
+	req: Request;
+	host: string;
+}): Promise<Headers> => {
+	// If Okta is disabled, always return the cookie header
+	const { useOkta } = await getOktaConfig();
+	console.log('useOkta', useOkta);
+	console.log(req.signedCookies[OAuthAccessTokenCookieName]);
+	if (!useOkta) {
+		return {
+			Cookie: getCookiesOrEmptyString(req),
+		};
+	}
+	switch (host) {
+		case 'members-data-api.' + conf.DOMAIN:
+			return {
+				Authorization: `Bearer ${req.signedCookies[OAuthAccessTokenCookieName]}`,
+			};
+		default:
+			return {};
+	}
+};
+
 export const customMembersDataApiHandler = proxyApiHandler(
```

Comment:
Could we use this function rather than having to export the authorization one?

Reply:
I had a go at this but it's quite hard to wrangle. Because it's a callback that doesn't return anything, we can give it our express Response object but we don't actually want to hijack what we return to the user